packages/components/src/components/progress-bar/progress-bar.css

```diff
@@ -0,0 +1,92 @@
+/**
+* @prop --progress-bar-display: Display of the progress bar
+* @prop --progress-bar-width: Width of the progress bar
+* @prop --progress-bar-align-items: Align items of the progress bar
+* @prop --progress-bar__outer-margin: Margin of the progress bar outer
+* @prop --progress-bar__outer-position: Position of the progress bar outer
+* @prop --progress-bar__outer-height: Height of the progress bar outer
+* @prop --progress-bar__outer-border-radius: Border radius of the progress bar outer
+* @prop --progress-bar__outer-background: Background of the progress bar outer
+* @prop --progress-bar__outer-overflow: Overflow of the progress bar outer
+* @prop --progress-bar__outer-position: Position of the progress bar outer
+* @prop --progress-bar__outer-width: Width of the progress bar outer
+* @prop --progress-bar__inner-position: Position of the progress bar inner
+* @prop --progress-bar__inner-left: Left of the progress bar inner
+* @prop --progress-bar__inner-top: Top of the progress bar inner
+* @prop --progress-bar__inner-height: Height of the progress bar inner
+* @prop --progress-bar__inner-border-radius: Border radius of the progress bar inner
+* @prop --progress-bar__inner-width: Width of the progress bar inner
+* @prop --progress-bar__inner-white-space: White space of the progress bar inner
+* @prop --progress-bar__inner-transition: Transition of the progress bar inner
+* @prop --progress-bar__inner-display: Display of the progress bar inner
+* @prop --progress-bar__inner-border-align-items: Align items of the progress bar inner
+* @prop --progress-bar__inner-justify-content: Justify content of the progress bar inner
+* @prop --progress-bar__inner-variant-success-background: Background of progress bar inner variant success
+* @prop --progress-bar__inner-variant-info-background: Background of progress bar inner variant info
+* @prop --progress-bar__inner-variant-warning-background: Background of progress bar inner variant warning
+* @prop --progress-bar__inner-variant-danger-background: Background of progress bar inner variant danger
+* @prop --progress-bar__inner-text-color: Text color of inside progress bar
+* @prop --progress-bar__inner-text-font-size: Text fond size of inside progress bar
+* @prop --progress-bar__inner-text-padding: Text padding of inside progress bar
+* @prop --progress-bar__text-color: Text color of progress bar
+* @prop --progress-bar__text-font-size: Text fond size of progress bar
+* @prop --progress-bar__text-display: Text display of progress bar
```

Comment:
we should stick to one naming convention: 
either `--{component-name}-{prop-name}` or `--{component-name}__{prop-name}`